interface/weno/validate.php

```diff
@@ -31,37 +34,37 @@
 $patient = $validation->validatePatient($pid);
 $pharmacy = $validation->patientPharmacyInfo($pid);
 
-if (empty($facility[0]['name']) || $facility[0]['name'] == "Your clinic name here") {
+if (empty($facility['name']) || $facility[0]['name'] == "Your clinic name here") {
```

Comment:
Doesn't the second conditional here also need to be fixed?

Reply:
yup.  PR updated.